stateMatrix.m

```diff
@@ -23,6 +25,9 @@
 RightPWM = round((100-TaskParameters.GUI.RightPokeAttenPrcnt) * 2.55);
 LEDErrorRate = 0.1;
 
+LeftSoundPWM = round((100-TaskParameters.GUI.LeftSoundAttenPrcnt) * 2.55);
+RightSoundPWM = round((100-TaskParameters.GUI.RightSoundAttenPrcnt) * 2.55);
+
```

Comment:
These 2 lines + lines 9 & 10 can all be moved down inside the `elseif` part at [line 50](https://github.com/ioannapandi/Mouse2AFC/pull/3/commits/bd06d2cd1af55ff7f37982979849b3ab734645aa#diff-d76dd70d2cab2ef003f54fa363676a010b0a03527768a021d94f10fbff5da7d6R50).
In a way we don't need to make this calculations, unless we are actually using these experiments types. It's not also much about the calculation power as much about grouping related code together as far as possible.

Reply:
I changed the commit and placed together all the "sounddiscrimination" related code inside the corresponding <!--elseif--> as you proposed! Let's discuss tomorrow the name of the rule and I'm willing to change all the parts in the code referred to the soundrule too (sound --> auditory).

Reply:
Sounds good, it would be nice to have Nassi's and Mostafa's input as well